### ISSUE TITLE
• Candy Shop

### ISSUE THREAD

**Issue by sleepyimoon** [ValveSoftware/Dota2-Gameplay#4667]
### Description

I wanted to add a new box of chocolates and eventually wrote off all the existing candies. There were 30 candies, now 0, I HAVE ZERO ON MY ACCOUNT.

**Comment by qubanacha** [comment]
Yes, before the update on November 3, for spraying only one bag of candy, but after it is given 6 bags, it's not fair!
